Daemaged.IBNet/Properties/AssemblyInfo.cs

```diff
@@ -1,35 +0,0 @@
-﻿using System.Reflection;
-using System.Runtime.InteropServices;
-
-// General Information about an assembly is controlled through the following
-// set of attributes. Change these attribute values to modify the information
-// associated with an assembly.
-[assembly: AssemblyTitle("Daemaged.IBNet")]
-[assembly: AssemblyDescription("IB TWS api for .NET")]
-[assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Dan Shechter")]
-[assembly: AssemblyProduct("IBNet")]
-[assembly: AssemblyCopyright("Copyright ©  2013")]
-[assembly: AssemblyTrademark("")]
-[assembly: AssemblyCulture("")]
-
-// Setting ComVisible to false makes the types in this assembly not visible
-// to COM components.  If you need to access a type in this assembly from
-// COM, set the ComVisible attribute to true on that type.
-[assembly: ComVisible(false)]
-
-// The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: Guid("2207b926-9637-4211-81e5-03914e5b2047")]
-
-// Version information for an assembly consists of the following four values:
-//
-//      Major Version
-//      Minor Version
-//      Build Number
-//      Revision
-//
-// You can specify all the values or you can default the Build and Revision Numbers
-// by using the '*' as shown below:
-// [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("9.6.8.12")]
```

Comment:
This needs to move to the IBNet.csproj so we still have working nupkgs

Reply:
Also, while you are at it, the .nuspec file needs to die (i.e. the content should be merged back to the csproj), as dotnet pack will generate the nuspec on the fly